Packs/PhishUp/Playbooks/PhishUp_Mail_Scanner.yml

```diff
@@ -0,0 +1,743 @@
+id: PhishUp Mail Scanner
+version: -1
+name: PhishUp Mail Scanner
+description: Extracts URLs from mail body and checks URLs with PhishUp. Takes action
+  based on PhishUp results.
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 628939c7-0e11-4b8a-85ce-e501d32b2a6c
+    type: start
+    task:
+      id: 628939c7-0e11-4b8a-85ce-e501d32b2a6c
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "1"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": -130
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: 8cb679ca-c36a-47b4-8608-56ef50d2576a
+    type: regular
+    task:
+      id: 8cb679ca-c36a-47b4-8608-56ef50d2576a
+      version: -1
+      name: Extracting URLs from Email
+      description: commands.local.cmd.extract.indicators
+      script: Builtin|||extractIndicators
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "12"
+    scriptarguments:
+      text:
+        simple: ${incident.labels.Email/html}
```

Comment:
Since some of the email integrations will provide different labels, it is better to use playbook input for all of the "labels" values which are being used in this playbook. So for example with this one, there's already a playbook input, we can just use it here instead of using the label. This will make the playbook more generic and users will be able to use it by mapping the values inside the mappers (or use existing mappers).

Reply:
We want to add more mail integration to our playbook, like you said other email integrations can has different labels for these reason we don't want to add this variables as playbook input